client/login/controller.js

```diff
@@ -159,3 +157,37 @@ export function redirectJetpack( context, next ) {
 	}
 	next();
 }
+
+export function setShouldSSRLogin( context, next ) {
+	if ( ! context.SSRValidators ) {
+		next(); // practically somehow the initSSRValidators was not initialized, so adding more is not necessary
+		return;
```

Comment:
I've been using `return next()` in cases like this.

Reply:
Not needed anymore, but I'll use return next() when I reach into a similar case.